<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>728</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>725</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/ProDent</normaloff>:/icons/ProDent</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background: url(:/images/bg2);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>25</number>
    </property>
    <property name="topMargin">
     <number>15</number>
    </property>
    <property name="rightMargin">
     <number>25</number>
    </property>
    <property name="bottomMargin">
     <number>25</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="widget_4" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="mainpageWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>800</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1282</width>
           <height>800</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image:none;</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>25</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <property name="styleSheet">
             <string notr="true">#widget_2 {
	background:#ffffff;
	border-radius:9px;
	border:1px solid #bbbbbb;
}</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="margin">
              <number>5</number>
             </property>
             <item>
              <widget class="QTextBrowser" name="textBrowser">
               <property name="styleSheet">
                <string notr="true">QTextBrowser {
	background:#ffffff;
	padding:10px;
	border:0px;
}

QScrollBar {
	background:none;
}</string>
               </property>
               <property name="html">
                <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Lucida Grande'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Welcome to ProMed!&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;What would you like to do?&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Appointments&lt;/span&gt;&lt;/p&gt;
&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;&lt;li style=&quot; font-size:10pt;&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;On your right, you will find your appointments for the day. By clicking on another day from the calendar, you can view the appointments of previous or following days.&lt;/li&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Create new patient record&lt;/span&gt;&lt;/p&gt;
&lt;li style=&quot; font-size:10pt;&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;To create a new patient record, click on the &amp;quot;New Patient&amp;quot; (&lt;img src=&quot;:/icons/New 32x32&quot; width=&quot;16&quot; /&gt;) button from the main toolbar.&lt;/li&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;List all patients&lt;/span&gt;&lt;/p&gt;
&lt;li style=&quot; font-size:10pt;&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;To view a list of all your patients, click on the &amp;quot;Patient List&amp;quot; (&lt;img src=&quot;:/icons/Patients 32x32&quot; width=&quot;16&quot; /&gt;) button, from the main toolbar. You can sort the list by clicking on the table column labels. Double-clicking on a patient row, will open their patient record.&lt;/li&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Appointments Calendar&lt;/span&gt;&lt;/p&gt;
&lt;li style=&quot; font-size:10pt;&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;To view the appointments for the whole month, as well as the previous and next months, with descriptions, time, and patient name, click on the &amp;quot;Appointments&amp;quot; (&lt;img src=&quot;:/icons/Calendar 32x32&quot; width=&quot;16&quot; /&gt;) button, from the main toolbar. Leaving your mouse pointer over the cell for each day, will pop up the details window for this day.&lt;/li&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Statistics&lt;/span&gt;&lt;/p&gt;
&lt;li style=&quot; font-size:10pt;&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;To view the statistics table of your dental office, click on the &amp;quot;Statistics&amp;quot; (&lt;img src=&quot;:/icons/Stats 32x32&quot; width=&quot;16&quot; /&gt;) button, from the main toolbar.&lt;/li&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Settings&lt;/span&gt;&lt;/p&gt;
&lt;li style=&quot; font-size:10pt;&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;To enter your medical office information in the system, change your login password, currencies, or languages, click on the &amp;quot;Settings&amp;quot; (&lt;img src=&quot;:/icons/Settings 32x32&quot; width=&quot;16&quot; /&gt;) button, from the main toolbar.&lt;/li&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Logout&lt;/span&gt;&lt;/p&gt;
&lt;li style=&quot; font-size:10pt;&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;To log out without quitting the application (= return to the login screen), click on the &amp;quot;Logout&amp;quot; (&lt;img src=&quot;:/icons/Logout 32x32&quot; width=&quot;16&quot; /&gt;) button, from the main toolbar.&lt;/li&gt;&lt;/ul&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;To get help, you can contact us at promed@re-compile.com via e-mail, or call us at +30 2752 0 26374, Monday to Friday, 9.00am - 18.00am GMT+2.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_3" native="true">
               <property name="styleSheet">
                <string notr="true">#widget_3, QLabel {
	background:#ffffff;
}

#widget_3 {
	border-top:1px dotted #aaaaaa;
}</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <property name="leftMargin">
                 <number>20</number>
                </property>
                <property name="rightMargin">
                 <number>20</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>60</width>
                    <height>60</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="pixmap">
                   <pixmap resource="resources.qrc">:/images/logo-64</pixmap>
                  </property>
                  <property name="scaledContents">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_6">
                  <item>
                   <widget class="QLabel" name="info_pname">
                    <property name="font">
                     <font/>
                    </property>
                    <property name="text">
                     <string>PName:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="info_pversion">
                    <property name="font">
                     <font/>
                    </property>
                    <property name="text">
                     <string>PVersion:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="info_plicense">
                    <property name="font">
                     <font/>
                    </property>
                    <property name="text">
                     <string>Licensed To:</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="homeWidgets" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QWidget" name="widget" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">#widget {
background:#ffffff;
border-radius:9px;
border:1px solid #bbbbbb;
}

QWidget {
background:#ffffff;
}

QToolButton {
	background:none;
	color:#000;
}</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QCalendarWidget" name="calendarWidget">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>200</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="navigationBarVisible">
                   <bool>true</bool>
                  </property>
                  <property name="dateEditEnabled">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="formHeader1">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">background:none;</string>
               </property>
               <property name="text">
                <string>Appointments for selected day:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="eventsWindow" native="true">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>305</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>305</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font/>
               </property>
               <property name="styleSheet">
                <string notr="true">#eventsWindow {
	background: #ffffff;
	border-radius: 9px;
	border:1px solid #bbbbbb;
}

#scrollAreaWidgetContents {
	background:#ffffff;
}

QScrollBar {
	background: none;
}

QLabel {
	background:#ffffff;
}</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QScrollArea" name="scrollArea">
                  <property name="frameShape">
                   <enum>QFrame::NoFrame</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Plain</enum>
                  </property>
                  <property name="lineWidth">
                   <number>1</number>
                  </property>
                  <property name="widgetResizable">
                   <bool>true</bool>
                  </property>
                  <widget class="QWidget" name="scrollAreaWidgetContents">
                   <property name="geometry">
                    <rect>
                     <x>0</x>
                     <y>0</y>
                     <width>281</width>
                     <height>271</height>
                    </rect>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_7">
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <layout class="QVBoxLayout" name="verticalLayout"/>
                    </item>
                   </layout>
                  </widget>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionFullscreen"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionHome"/>
   <addaction name="separator"/>
   <addaction name="actionNew"/>
   <addaction name="separator"/>
   <addaction name="actionList_Patients"/>
   <addaction name="separator"/>
   <addaction name="actionAppointments"/>
   <addaction name="separator"/>
   <addaction name="actionStatistics"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
   <addaction name="separator"/>
   <addaction name="actionBackup"/>
   <addaction name="separator"/>
   <addaction name="actionE_syntagografisi"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionQuit"/>
   <addaction name="separator"/>
   <addaction name="actionLogout"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normalon>:/icons/New 32x32</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>New Patient</string>
   </property>
   <property name="toolTip">
    <string>Create a new patient record</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionNew_2">
   <property name="text">
    <string>New...</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Quit 32x32</normaloff>:/icons/Quit 32x32</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>quit this program</string>
   </property>
  </action>
  <action name="actionList_Patients">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Patients 32x32</normaloff>:/icons/Patients 32x32</iconset>
   </property>
   <property name="text">
    <string>Patient List</string>
   </property>
   <property name="toolTip">
    <string>display a list of the patients in system</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="icon">
    <iconset>
     <normalon>:/icons/Home 32x32</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
   <property name="toolTip">
    <string>Back to Home Page</string>
   </property>
  </action>
  <action name="actionStatistics">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Stats 32x32</normaloff>:/icons/Stats 32x32</iconset>
   </property>
   <property name="text">
    <string>Statistics</string>
   </property>
   <property name="toolTip">
    <string>displays the statistics chart of your dental office</string>
   </property>
  </action>
  <action name="actionAppointments">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Calendar 32x32</normaloff>:/icons/Calendar 32x32</iconset>
   </property>
   <property name="text">
    <string>Appointments</string>
   </property>
   <property name="toolTip">
    <string>Appointments in month view</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Settings 32x32</normaloff>:/icons/Settings 32x32</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>application settings</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Logout 32x32</normaloff>:/icons/Logout 32x32</iconset>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
   <property name="toolTip">
    <string>Logout of the system and back to Login screen</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
   <property name="toolTip">
    <string>About This Application</string>
   </property>
  </action>
  <action name="actionBackup">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Backup 32x32</normaloff>:/icons/Backup 32x32</iconset>
   </property>
   <property name="text">
    <string>Backup</string>
   </property>
   <property name="toolTip">
    <string>Internet Backup of the System</string>
   </property>
  </action>
  <action name="actionE_syntagografisi">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Browser 32x32</normaloff>:/icons/Browser 32x32</iconset>
   </property>
   <property name="text">
    <string>e-syntagografisi</string>
   </property>
   <property name="toolTip">
    <string>e-syntagografisi link</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fullscreen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
